ecosystem/node/run-validator.mdx

```diff
@@ -0,0 +1,670 @@
+
+### 3.1 Get validator wallet address
+
+```sh
+MyTonCtrl> wl
+```
+
+Find the wallet named `validator_wallet_001` and its address.
+
+<Image
+  src="/resources/images/nodes/run-validator/run-validator-7.png"
+  darkSrc="/resources/images/nodes/run-validator/run-validator-7.png"
+  alt="Print validator wallet list"
+/>
+
+### 3.2 Back-up validator wallet secret key
+
+<Aside
+  type="caution"
+>
+  This private key is required to restore access to the funds stored in the validator wallet. It is not involved in managing rewards or staking.
+</Aside>
+
+Make a backup of the validator wallet secret key:
+
+```sh
+MyTonCtrl> ew validator_wallet_001
+```
+
+<Image
+  src="/resources/images/nodes/run-validator/run-validator-11.png"
+  darkSrc="/resources/images/nodes/run-validator/run-validator-11.png"
+  alt="Back-up validator wallet"
+/>
+
+<Aside type="tip">
+  Import this secret key with [`iw`](/ecosystem/node/mytonctrl/wallet#iw) command.
+</Aside>
+
+### 3.3 Explore validator wallet
+
+Explore validator wallet by address with [Tonviewer](https://tonviewer.com/):
+
+<Image
+  src="/resources/images/nodes/run-validator/run-validator-6.png"
+  darkSrc="/resources/images/nodes/run-validator/run-validator-6.png"
+  alt="Explore validator wallet"
+/>
+
+### 3.4 Credit validator wallet
+
+<Aside
+  type="danger"
+  title="Funds at risk"
+>
+  Ensure precise handling of operations, carefully verifying the addresses for `single_nominator_pool`, `owner_address`, and `validator_wallet`. Any incorrect deposit processing may result in the permanent and unrecoverable loss of funds.
+</Aside>
+
+Credit the validator wallet. Ensure that at least 200 TON per month is available in the validator’s wallet to cover operational fees.
+
+<Image
+  src="/resources/images/nodes/run-validator/run-validator-9.png"
+  darkSrc="/resources/images/nodes/run-validator/run-validator-9.png"
+  alt="Credit validator wallet"
+/>
+
+### 3.5 Activate the wallet
+
+Activate (deploy on-chain) validator wallet, run:
+
+```sh
+MyTonCtrl> aw validator_wallet_001
+```
+
+<Image
+  src="/resources/images/nodes/run-validator/run-validator-12.png"
+  darkSrc="/resources/images/nodes/run-validator/run-validator-12.png"
+  alt="Activate validator wallet command result"
+/>
+
+That also reflects in Tonviewer; the status will be displayed as `Active`
+
+<Image
+  src="/resources/images/nodes/run-validator/run-validator-14.png"
+  darkSrc="/resources/images/nodes/run-validator/run-validator-14.png"
+  alt="Validator wallet in explorer"
+/>
+
+### 3.6 Create a pool
+
+Create a [single nominator pool](https://docs.ton.org/v3/documentation/smart-contracts/contracts-specs/single-nominator-pool) for secure stake management. As the `owner-address`, specify the beneficiary wallet address that will stake the owner's funds and receive rewards.
```

Comment:
What is `owner-address`? Should describe prerequisites (set up cold wallet, etc.)